systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthAuthorizationST.java

```diff
@@ -375,7 +369,7 @@ void testSuperUserWithOauthAuthorization(ExtensionContext extensionContext) {
             .build();
 
         resourceManager.createResource(extensionContext, teamAOauthClientJob.consumerStrimziOauthTls(oauthClusterName).build());
-        assertThrows(WaitException.class, () -> JobUtils.waitForJobFailure(teamAConsumerName, NAMESPACE, 30_000));
+        assertThrows(WaitException.class, () -> JobUtils.waitForJobFailure(teamAConsumerName, 30_000));
```

Comment:
I wonder if we should keep the namespace a use it properly instead of this. It could open a way to flaky tests because of test parallelism.

Reply:
@see-quick ^?

Reply:
The sollution on this problem is to re-write method to use that `NAMESPACE` instance variable. 
```
Inside method is:
 !kubeClient().checkSucceededJobStatus(jobName))
```
and this could provide erroes because of wrong set namespace during test paralelism. The correct way how to solve this:
```
kubeClient(namespace).checkSucceededJobStatus(namespace, jobName, 1)
```

Reply:
So who should fix it/where? In this PR or in next PR?

Reply:
Now I guess